.github/workflows/php.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Unit tests
```

Comment:
In general, I'd add the same comments as in #38 -- maybe put this on the back burner, until #38 is resolved?